nntrainer/layers/layer_node.cpp

```diff
@@ -307,9 +308,8 @@ const std::vector<std::string> LayerNode::getInputLayers() const {
   names.reserve(input_connections.size());
   std::transform(
     input_connections.begin(), input_connections.end(),
-    std::back_inserter(names), [](const Connection &con) -> const auto & {
-      return con.getName();
-    });
+    std::back_inserter(names),
+    [](const Connection &con) -> const auto & { return con.getName(); });
```

Comment:
no need to modify.

Reply:
It may be changed by clang-format checker;;